tag 101 :
  id : 30306
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.1
tag 102 :
  id : 30270
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 103 :
  id : 30269
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 104 :
  id : 30226
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 105 :
  id : 30262
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 106 :
  id : 30264
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 107 :
  id : 30313
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 108 :
  id : 30246
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 109 :
  id : 30259
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 110 :
  id : 30266
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 111 :
  id : 30297
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 112 :
  id : 30305
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 113 :
  id : 30252
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 114 :
  id : 30258
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 115 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 116 :
  id : 30267
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 117 :
  id : 30444
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 118 :
  id : 26715
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 119 :
  id : 26627
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 120 :
  id : 26643
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 121 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 122 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 123 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 124 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 125 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 126 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 127 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 128 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 129 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 130 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 131 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 132 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 133 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 134 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 135 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 136 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 137 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 138 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 139 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 140 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 141 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 142 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 143 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 144 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 145 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 146 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 147 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 148 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 149 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001
tag 150 :
  id : 30307
  x_alpha : 0.01
  y_alpha : 0.01
  z_alpha : 0.001